addons/website/static/src/snippets/s_table_of_content/000.js

```diff
@@ -13,7 +13,7 @@ const TableOfContent = publicWidget.Widget.extend({
      */
     async start() {
         await this._super(...arguments);
-        this.$scrollingElement = $().getScrollingElement();
+        this.$scrollingElement = this.$target.closest(".s_table_of_content").closestScrollable();
```

Comment:
Should probably have used our util which is a shortcut / in charge of calling jQuery directly 
```suggestion
        this.$scrollingElement = dom.closestScrollable(this.$target[0].closest(".s_table_of_content"));
```
So when we remove jQuery, we adapt the `closestScrollable` from `dom` and it works without having to change all lines.
But it's ok as there is a mix usage of both notation in the code.